docs/Systems/Tailwind/UtilityClassReferences/Effects/Shadow.stories.tsx

```diff
@@ -16,18 +15,17 @@ const classEntries: Array<{ utilityClassName: string; cssProperty: string }> =
   )
 
 export default {
-  title: `${CATEGORIES.tailwind}/Utility Class References/Effects/Box Shadow`,
-  component: <div>Hello</div>,
+  title: "Systems/Tailwind/Utility Class References/Effects/Box Shadow",
   parameters: {
     docs: {
       description: {
         component: utilityDescription(prefix, classEntries[0].utilityClassName),
       },
     },
   },
-}
+} as Meta
```

Comment:
An unfortunate _meta_ that we need to use `as` here

Reply:
Actually, we might be able to use `satisfies` now :O 
Note to self to make the change :3